troubleshooting/README.md

```diff
@@ -1,6 +1,46 @@
-Run
+# Troubleshooting
+
+### Log collector script
+
+The log collector script will collect  
+- output of describe pod
+- yaml spec of pod
+- efs-utils logs from /var/log/amazon/efs
+- efs-utils state files from /var/run/efs
+- nfs mounts on filesystem
+
```

Comment:
https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/troubleshooting/log_collector.py#L33-L45
 I think we should list the kubectl CLIs to make customers more clear instead of the description here
 
 Specifically `- output of describe pod
- yaml spec of pod`

Reply:
done